pylds/base_discrete.py

```diff
@@ -26,6 +27,109 @@ def check_if_points_escape_box(u, box_boundaries):
     u_indices = (x >= box_x_min) & (x <= box_x_max) & (y >= box_y_min) & (y <= box_y_max)
     return u_indices
 
+def pbc_correction_coords_single_axis(x, box_origin, box_length):
+    """
+    Correct coordinate in a single axis by Periodic Boundary Conditions (PBCs).
+
+    Parameters
+    ----------
+    x : array_like, shape(n, )
+        points in axis to correct by PBCs
+    
+    origin : array
+        
+    box_length : array
+        
+    Returns
+    -------
+    x_pbc : array_like, shape(n, )
+        array of corrected points for periodic box
+    """
+    x_pbc = x
+    x0 = box_origin
+    L = box_length
+    if not isinstance(x0, bool) or not isinstance(L, bool):
```

Comment:
Following the docstring x0 are L arrays (of floats?). Why would any of them be boolean?

Reply:
Yes. If the correction for PBC can be done for one single axis, e.g., when PBC on a cylinder. 
For example, for `periodic_boundaries = [(0, L), (False, False)]` the correction will be only for the X-axis, is the second tuple has at least one entry as False. When no PBC, here the Y-axis, then coordinates are left without correction.

Reply:
As long as it's clear from the dosctring, it's ok